File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.rs
impl Solution {
    pub fn remove_vowels(s: String) -> String {
        let mut ans = String::new();
        for x in s.chars()
        {
            if x != 'a' && x != 'e' && x != 'i' && x != 'o' && x != 'u'
            {
                ans.push(x);
            }
        }
        ans
    }
}

File: 1119-remove-vowels-from-a-string/1119-remove-vowels-from-a-string.cpp
class Solution {
public:
    string removeVowels(string s) {
    string ans;
    set<char> vowels {'a', 'e', 'i', 'o', 'u'};
    for (auto c : s)
        if (vowels.find(c) == vowels.end())
            ans += c;
    return ans;
}
};